examples/hello.rs

```diff
@@ -22,7 +28,8 @@ fn main() -> Result<(), Box<dyn Error>> {
     let mut back_buffer = surface.back_buffer().unwrap();
 
     'app: loop {
-        for event in surface.poll_events() {
+        // FIXME: This doesn't seem to work for window events on linux/wayland(mutter)
```

Comment:
Currently I don't get `WindowEvent`s in the `hello` example on Linux/Wayland(Mutter).
Don't know if this is specific to my configuration or universal.

Reply:
I am having the same issue on Linux/X11. I'm guessing this has something to do with the change to the `events_rx` queue. I will open an issue for luminance_glfw.

Reply:
The `poll_events` call is missing here. See [this](https://github.com/phaazon/luminance-rs/blob/master/luminance-examples/src/hello-world.rs#L210-L211) for further details. :)

Reply:
fixed in f75eb6d2